src/main/java/com/example/codefellowship/database/AppUser.java

```diff
@@ -0,0 +1,115 @@
+package com.example.codefellowship.database;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@Entity
+public class AppUser implements UserDetails {
+
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @OneToMany(mappedBy = "AppUser")
```

Comment:
The mapping here should match the variable name you used for your mapping in the Post class.